app/models/framework/definition/RM3756.rb

```diff
@@ -6,56 +6,107 @@ class RM3756 < Base
-        field 'Pro-Bono Price per Unit', :decimal, exports_to: 'Additional1'
-        field 'Pro-Bono Quantity', :decimal, exports_to: 'Additional2'
-        field 'Pro-Bono Total Value', :decimal, exports_to: 'Additional3'
-        field 'Sub-Contractor Name (If Applicable)', :string, exports_to: 'Additional4'
+        field 'Matter Name', :string, exports_to: 'CustomerReferenceNumber', presence: true
+        field 'Pricing Mechanism', :string, exports_to: 'Additional5', presence: true, case_insensitive_inclusion: { in: PRICING_MECHANISM_VALUES }
+        # field 'Award Procedure', :string, exports_to: 'Additional6', presence: true
+        field 'Pro-Bono Price per Unit', :string, exports_to: 'Additional1', ingested_numericality: true
+        field 'Pro-Bono Quantity', :string, exports_to: 'Additional2', ingested_numericality: true
+        field 'Pro-Bono Total Value', :string, exports_to: 'Additional3', ingested_numericality: true
+        field 'Sub-Contractor Name (If Applicable)', :string, exports_to: 'Additional4', presence: true
       end
 
       class Order < Sheet
         total_value_field 'Expected Total Order Value'
 
-        field 'Tier Number', :string, exports_to: 'LotNumber'
-        field 'Supplier Reference Number', :string, exports_to: 'SupplierReferenceNumber'
-        field 'Customer URN', :integer, exports_to: 'CustomerURN'
-        field 'Customer Organisation Name', :string, exports_to: 'CustomerName'
-        field 'Customer Post Code', :string, exports_to: 'CustomerPostCode'
-        field 'Matter Name', :string, exports_to: 'CustomerReferenceNumber'
+        field 'Tier Number', :string, exports_to: 'LotNumber', presence: true, inclusion: { in: %w[1 2] }
+        field 'Supplier Reference Number', :string, exports_to: 'SupplierReferenceNumber', presence: true
+        field 'Customer URN', :integer, exports_to: 'CustomerURN', urn: true
+        field 'Customer Organisation Name', :string, exports_to: 'CustomerName', presence: true
+        field 'Customer Post Code', :string, exports_to: 'CustomerPostCode', presence: true
+        field 'Matter Name', :string, exports_to: 'CustomerReferenceNumber', presence: true
         field 'Matter Description', :string, exports_to: 'ProductDescription'
-        field 'Contract Start Date', :date, exports_to: 'ContractStartDate'
-        field 'Contract End Date', :date, exports_to: 'ContractEndDate'
-        field 'Award Procedure', :string, exports_to: 'ContractAwardChannel'
-        field 'Expected Total Order Value', :decimal, exports_to: 'ContractValue'
+        field 'Contract Start Date', :string, exports_to: 'ContractStartDate'
+        field 'Contract End Date', :string, exports_to: 'ContractEndDate'
+        field 'Award Procedure', :string, exports_to: 'ContractAwardChannel', presence: true, case_insensitive_inclusion: { in: ['Further Competition', 'Direct Award'] }
+        field 'Expected Total Order Value', :string, exports_to: 'ContractValue'
```

Comment:
Why do ContractValue, ContractEndDate, ContractStartDate and Expected Pro-Bono value not get any validation?